src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
```

Comment:
Cash is a small jQuery alternative that provides jQuery-style syntax for manipulating the DOM. Using this tool in our application facilitates writing our code due to the simplicity of syntax provided by Cash. However, it would not be a bad idea to use it via CDN in order to reduce production build size.